src/main/resources/default/templates/biz/model/query-results.html.pasta

```diff
@@ -4,61 +4,100 @@
 <i:arg type="List" name="entities"/>
 <i:arg type="List" name="properties"/>
 
-<w:page title="Query Tool">
+<t:page title="Query Tool">
     <i:block name="breadcrumbs">
         <li><a href="/system/query">Query Tool</a></li>
     </i:block>
 
-    <w:pageHeader title="Query Tool"/>
+    <t:pageHeader title="Query Tool"/>
 
     <i:invoke template="/templates/biz/model/query-header.html.pasta" query="@query" limit="@limit" type="type"/>
 
-    <i:for type="sirius.db.mixing.BaseEntity" var="entity" items="entities">
-        <div class="well">
-            <table class="table table-bordered table-striped">
-                <tr>
-                    <td colspan="2" class="align-center"><b>@type.getType().getSimpleName(): @entity.getIdAsString()</b></td>
-                </tr>
-                <i:for type="sirius.db.mixing.Property" var="property" items="properties">
-                    <tr>
-                        <td clas="col-md-3">@property.getName()</td>
-                        <td clas="col-md-9">
-                            <i:if test="property.is(sirius.db.mixing.properties.LocalDateTimeProperty.class)
-                                 || property.is(sirius.db.mixing.properties.LocalDateProperty.class)
-                                 || property.is(sirius.db.mixing.properties.IntegerProperty.class)
-                                 || property.is(sirius.db.mixing.properties.LongProperty.class)
-                                 || property.is(sirius.db.mixing.properties.BooleanProperty.class)">
-                                <div class="align-right">@toUserString(property.getValue(entity))</div>
-                                <i:else>
+    <t:emptyCheck data="@entities">
+        <t:datacards size="x-large">
+            <i:for type="sirius.db.mixing.BaseEntity" var="entity" items="entities">
+                <t:datacard title="@entity.getIdAsString()" subTitle="@entity.getClass().getName()">
+                    <table class="table table-hover">
```

Comment:
In my opinion the `text-small` improved the usability of the Query Tool quite a bit as more properties / information is shown at once.